client-v2/src/components/FrontsEdit/ArticleFragmentForm.tsx

```diff
@@ -357,18 +359,45 @@ class FormComponent extends React.Component<Props, FormComponentState> {
           <RowContainer>
             <Row>
               <Col>
-                <ImageWrapper faded={imageHide}>
-                  <ConditionalField
-                    permittedFields={editableFields}
-                    name="primaryImage"
-                    component={InputImage}
-                    disabled={imageHide}
-                    criteria={imageCriteria}
-                    frontId={frontId}
+                {imageReplace && (
+                  <ImageWrapper faded={imageHide}>
+                    <ConditionalField
+                      permittedFields={editableFields}
+                      name="primaryImage"
+                      component={InputImage}
+                      disabled={imageHide}
+                      criteria={imageCriteria}
+                      frontId={frontId}
+                    />
+                  </ImageWrapper>
+                )}
+                {!imageReplace && (
+                  <ThumbnailEditForm
+                    style={{
+                      backgroundImage: `url('${
+                        articleCapiFieldValues.thumbnail
+                      }')`,
+                      opacity: imageHide ? 0.5 : 1
+                    }}
                   />
-                </ImageWrapper>
+                )}
               </Col>
               <Col>
+                <InputGroup>
+                  <ConditionalField
+                    permittedFields={editableFields}
+                    name="imageReplace"
+                    component={InputCheckboxToggle}
+                    label="Replace media"
+                    id={getInputId(articleFragmentId, 'image-replace')}
+                    type="checkbox"
+                    default={false}
+                    onChange={e => {
+                      change('imageHide', false);
+                      change('imageReplace', true);
```

Comment:
This is to have similar behaviour in the old and new tool and avoid weird behaviours if people switch between the two.